src/Views/AddDog/AddDog.js

```diff
@@ -0,0 +1,55 @@
+import { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import DogForm from '../../Comps/DogForm';
+import { addDog } from '../../services/fetchdogs';
+//update
+
+export default function AddDog() {
+
+  const [img, setImg] = useState('');
+  const [Name, setName] = useState('');
+  const [Age, setAge] = useState('');
+  const [Breed, setBreed] = useState('');
+  const [Desc, setDesc] = useState('');
+  const [error, setError] = useState('');
+  const [message, setMessage] = useState('');
+  
+
+  const history = useHistory();
+  
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      await addDog({ Name, Age, Desc, Breed, img });
+      setMessage('Added Good');
+      history.push('/');
```

Comment:
Here we will want to call a setTimeout function so the user can see the message before being redirected to home.